src/routes/events/+page.svelte

```diff
@@ -36,10 +67,55 @@
 <Container>
 	<div class="flex flex-col mt-8 md:mt-16">
 		<section class="flex z-20 flex-wrap justify-center md:mt-0 gap-8 md:gap-28">
-			{#each data.meetups as meetup (meetup.title)}
+			{#each paginatedEvents as meetup (meetup.title)}
 				<EventCard {meetup} />
 			{/each}
 		</section>
+		
+		<!-- Pagination controls -->
+		{#if totalPages > 1}
+			<div class="flex justify-center items-center mt-12 mb-8 space-x-2">
+				<button 
+					on:click={prevPage} 
```

Comment:
Since `on:click` is deprecated, we can use `onclick` instead.
```suggestion
					onclick={prevPage} 
```

Reply:
updated ✅